### PR TITLE
Dp 315 - Try to add required fields to the connector and show in two different divs

### PR DESCRIPTION
**Pull request by alex-504**

### CODE REVIEW

File: app/admin_components/adf-services/dreamfactory-services.js [hunk 1]
```diff
@@ -463,7 +463,7 @@ angular.module('dfServices', ['ngRoute', 'dfUtility', 'dfApplication'])
                 }
 
                 scope.saveOrUpdateService = function () {
-
+                    console.log(scope.serviceInfo);
```
Comment (tomonorman) [inline app/admin_components/adf-services/dreamfactory-services.js:466]: @alex-504 Can you delete this console.log?

Comment (alex-504) [reply app/admin_components/adf-services/dreamfactory-services.js]: Sure. Done!

File: app/admin_components/adf-services/views/df-service-config.html [hunk 2]
```diff
@@ -1,410 +1,9 @@
...
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-    </div>
+    <div ng-if="selectedSchema.name === 'mysql'" ng-include="'admin_components/adf-services/views/df-service-mysql.html'"></div>
+    <div ng-if="selectedSchema.name === 'sqlsrv'" ng-include="'admin_components/adf-services/views/df-service-sqlsrv.html'"></div>
+    <div ng-if="selectedSchema.name === 'oracle'" ng-include="'admin_components/adf-services/views/df-service-oracle.html'"></div>
+    <div ng-if="selectedSchema.name === 'pgsql'" ng-include="'admin_components/adf-services/views/df-service-pgsql.html'"></div>
```
Comment (yaroslavmo) [inline app/admin_components/adf-services/views/df-service-config.html:5]: Nothing would change if we write
`<div 
ng-if="selectedSchema.name === 'mysql' || selectedSchema.name === 'sqlsrv' || selectedSchema.name === 'oracle' || selectedSchema.name === 'pgsql' " 
ng-include="'admin_components/adf-services/views/df-service-pgsql.html'">`

or am I wrong?

File: app/admin_components/adf-services/dreamfactory-services.js [hunk 3]
```diff
@@ -1487,12 +1486,22 @@ angular.module('dfServices', ['ngRoute', 'dfUtility', 'dfApplication'])
                         configObj[key] = [];
                     }
 
+                    angular.forEach(
+                        function (field) {
+                            if (field.required) {
+                                scope.serviceField = true;
```
Comment (yaroslavmo) [inline app/admin_components/adf-services/dreamfactory-services.js:1492]: If the variable is boolean, the name should start with `is` or `has`. 

Also, I don't think serviceField is ever used anywhere.

**Review by yaroslavmo — CHANGES_REQUESTED** [review]

File: app/admin_components/adf-services/dreamfactory-services.js (hunk 3 above)
Comment (yaroslavmo) [reply app/admin_components/adf-services/dreamfactory-services.js:1492]: Remove

**Review by yaroslavmo — CHANGES_REQUESTED** [review]

**Comment by yaroslavmo** [comment]
Use DP-315-test branch instead

**Pull request closed without merging**